docker-compose.yml

```diff
@@ -26,28 +26,40 @@ services:
         networks:
             - localdev
     npm:
-        image: postgres-django
+        image: hedera-django
         build:
             context: .
             dockerfile: Dockerfile
+            target: prod
         container_name: hedera-npm
-        volumes:
-            - ./:/app
         working_dir: /app
         networks:
             - localdev
         ports:
             - "8080:8080"
         command: /bin/sh -c 'npm i && npm start'
     django:
-        image: postgres-django
+        image: hedera-django-dev
         build:
             context: .
             dockerfile: Dockerfile
+            target: dev
         container_name: hedera-django
         working_dir: /app
         volumes:
-            - ./:/app
+            - ./cms/:/app/cms
+            - ./fixtures/:/app/fixtures
+            - ./groups/:/app/groups
+            - ./hedera/:/app/hedera
+            - ./import-data/:/app/import-data
+            - ./lattices/:/app/lattices
+            - ./lemmatization/:/app/lemmatization
+            - ./lemmatized_text/:/app/lemmatized_text
+            - ./lti/:/app/lti
+            - ./pdfservice/:/app/pdfservice
+            - ./static/src/:/app/static/src
+            - ./vocab_list/:/app/vocab_list
+            - ./manage.py/:/app/manage.py
```

Comment:
I'm not a big fan of this long, needs-to-be-maintained list of mappings. It's important because mapping `node_modules` isn't a good idea between OSX and Ubuntu -- I've observed the container django blow up if it's accessing `node_modules` installed for OSX. I'm no expert here, so suggestions welcome re: `node_modules` issues.

Further, this doesn't include all the `load_*.py` scripts that are referenced in the README. If we settle on this approach, we can maybe move those into a `scripts/` dir to make it easier to include here; maybe we'll come up with a better approach to host-container mapping for development, though, and avoid that issue somehow! :)

Reply:
We are planning to consolidate those `load_*.py` scripts in another story so that might become easier to manage.

Reply:
Wonder if we could use something like [extends configuration](https://docs.docker.com/compose/extends/#understand-the-extends-configuration) to extend the volumes into a seperate yml file just to keep them DRY.

Reply:
@arthurian interesting -- I think in this case I favor explicit, because getting tricky with volumes could lead to subtle issues which could be harder for us, let alone casual contributors, to troubleshoot. I will explore the `extends` mod @Tanvez proposed to see if it'll make documenting/maintaining clearer, though!